src/pixel.js

```diff
@@ -59,7 +59,9 @@ function createNoReferrerPixel(win, src) {
       })
     );
     win.document.body.appendChild(iframe);
-    createImagePixel(iframe.contentWindow, src);
+    iframe.onload = function() {
```

Comment:
please register the onload event before appending iframe to the document in case the onload event fire beforehand. Thanks

Reply:
Done!